export/index.ts

```diff
@@ -239,7 +239,8 @@ class NeonPool extends Pool {
 
       const sql = neon(connectionString, { fullResults: true, arrayMode: config.rowMode === 'array' });
 
-      sql(queryText, queryValues)
+      // @ts-ignore -- TS doesn't know about this.options
```

Comment:
```suggestion
      // @ts-expect-error -- TS doesn't know about this.options
```

Reply:
Also I tried following the code to see how we have support for this last `options` parameter and could just barely follow.

I wonder if we can improve the types here instead?

But sure as a fix this is good enough, especially since it works 😄

Reply:
I agree, this is a bit of a bodge: it might be good to revisit in future.